server/routes/notification.js

```diff
@@ -138,13 +141,15 @@ router.get(
           csrfToken: res.locals.csrfToken,
           hmppsAuthMFAUser: req.hmppsAuthMFAUser,
           authUrl: req.authUrl,
+          isSnoozed: moment(preferences.snoozeUntil).isAfter(moment()),
+          snoozeUntil: moment(preferences.snoozeUntil).format('dddd, Do MMMM, YYYY'),
```

Comment:
Surely we don't need two values here? `isSnoozed` is implied by `snoozeUntil`.
You could extract this out to a utility and test separately then if you wanted then?
*pseudo code alert* 😬
```
const getTimeUntil = date => {
  const testMoment = moment(date)
  return testMoment.isAfter(moment()) ? 
    testMoment.format('dddd, Do MMMM, YYYY') : ''
}
```